ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/DC1_FABRIC.yml

```diff
@@ -246,6 +250,20 @@ l2leaf:
     rack: "rackE"
     mlag_peer_l3_ipv4_pool: 10.255.251.0/24
     mlag_peer_ipv4_pool: 10.255.252.0/24
+    uplink_structured_config:
+      storm_control:
+        broadcast:
+          level: 25
+        unknown_unicast:
+          level: 25
+      native_vlan: tag
+    mlag_structured_config:
+      storm_control:
+        broadcast:
+          level: 25
+        unknown_unicast:
+          level: 25
+      native_vlan: tag
```

Comment:
same here

Reply:
This is indeed a valid command, as per our chat, let's create a PR to update the documentation in `eos_cli_config_gen`.